tools/peptideshaker/peptide_shaker.xml

```diff
@@ -36,6 +36,18 @@
             -id_params \$cwd/SEARCHGUI_IdentificationParameters.par
             -out \$cwd/peptideshaker_output.cpsx
             -zip \$cwd/peptideshaker_output.zip
+            #set $cleaned_list = str($outputs).split(',')
```

Comment:
@CarlosHorro does this not work anymore? https://github.com/CarlosHorro/tools-galaxyp/blob/25f59f61cf5e089a2c456a4a2b89779507246b93/tools/peptideshaker/peptide_shaker.xml#L129

Reply:
Sorry I'm not sure if I understand the comment... I think it still works :-) do you mean that the following code (related to the 'cleaned_list') can be redundant?

Reply:
I think it is not needed as we filter the other strings already out. Do you see that this is not working without your code?

Reply:
Yes, it gan give an error. As an example, if you choose only "cps" and "PSM report", it gives and error as the "if" condition is true but there is still the "cps" option which need to be cleaned before showing it with the "-reports" option.

Reply:
Could you add a test for this?